datetime/example_test.go

```diff
@@ -0,0 +1,56 @@
+// Run Tarantool instance before example execution:
+// Terminal 1:
+// $ cd datetime
+// $ TEST_TNT_LISTEN=3013 TEST_TNT_WORK_DIR=$(mktemp -d -t 'tarantool.XXX') tarantool config.lua
+//
+// Terminal 2:
+// $ cd datetime
+// $ go test -v example_test.go
+package datetime_test
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/tarantool/go-tarantool"
+	_ "github.com/tarantool/go-tarantool/datetime"
+)
+
+// Example demonstrates how to use tuples with datetime. To enable support of
+// datetime import tarantool/datetime package.
+func Example() {
+	opts := tarantool.Opts{
+		User: "test",
+		Pass: "test",
+	}
+	conn, _ := tarantool.Connect("127.0.0.1:3013", opts)
```

Comment:
Maybe it would be better to check for errors in example_test.go?

Reply:
I believe it would not better. Because it is an example and not a production source code. I want to keep example brief and concise.

Reply:
It makes sense, but you check errors in `example_test.go` #146 . And we check errors in the [actual code](https://github.com/tarantool/go-tarantool/blob/master/example_test.go#L53).